lifetiles-graph/src/main/java/nl/tudelft/lifetiles/graph/controller/GraphController.java

```diff
@@ -329,6 +344,7 @@ private void repaint() {
 
             repaintPosition(scrollPane.hvalueProperty().doubleValue());
         }
+        getMiniMapController().drawMiniMap();
```

Comment:
The (re)drawing of the minimap should happen iff the graph gets repainted, so your suggestion would introduce redundant additional complexity.